marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/updateOptions.sjs

```diff
@@ -0,0 +1,24 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const updateOptionsLib = require("/data-hub/5/data-services/mastering/updateOptionsLib.sjs");
```

Comment:
I go back and forth on making a library vs just tossing all the code into the endpoint, and waiting until the need arises to move it into a library for reuse. Thoughts on that? I assume we'll revisit this when we handle merging steps.

Reply:
Personally, I like the library approach. Both match and merge could use the same library and then if we anything other than the data service needs it, we can always move it into another folder.